helper.fish

```diff
@@ -1383,8 +1383,9 @@ function checkLogId
   or begin popd; return 1; end
 
   set -l ids (find lib arangod arangosh enterprise -name "*.cpp" -o -name "*.h" \
-    | xargs grep -h 'LOG_\(TOPIC\|TRX\|TOPIC_IF\)("[a-f0-9]*"' \
-    | sed -e 's:^.*LOG_[^(]*("\([a-f0-9]*\)".*:\1:')
+    | xargs grep -h 'LOG_\(TOPIC\|TRX\|TOPIC_IF\)("[a-z0-9]*"' \
+    | grep -v 'LOG_DEVEL' \
+    | sed -e 's:^.*LOG_[^(]*("\([a-z0-9]*\)".*:\1:')
```

Comment:
Would advise to use `| sed -e 's:^.*LOG_[^(]*("\([^\"]*\)".*:\1:')` here.

Reply:
The regex parts on line 1386 and 1388 just need to be the same.
It won't help changing only line 1388 without changing line 1386. So we need to make the change in both lines.